js/utils.js

```diff
@@ -500,7 +500,7 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
         }
 
         if (condition === 'oneof') {
-          return splitByCommas(filter.value).includes(rowData);
+          return Boolean(_.intersectionWith(splitByCommas(filter.value), splitByCommas(rowData), _.isEqual).length);
```

Comment:
```suggestion
          return !!_.intersectionWith(splitByCommas(filter.value), splitByCommas(rowData), _.isEqual).length;
```

Reply:
FIxed.

Reply:
Also, added fix for filter values in double quotes, they didn't work as expected because of comma splitting of the row data.